cmd/gdeploy/commands/notifications/slack/configure.go

```diff
@@ -56,16 +56,18 @@ var configureSlackCommand = cli.Command{
 			}
 		}
 
-		for _, v := range cfg {
-			err := deploy.CLIPrompt(v)
-			if err != nil {
-				return err
-			}
+		// for _, v := range cfg {
+		// 	err := deploy.CLIPrompt(v)
+		// 	if err != nil {
+		// 		return err
+		// 	}
+		// }
+
```

Comment:
Can delete this?

Reply:
Left commented out by mistake, I have reverted thanks